Mathlib/NumberTheory/PrimeCounting.lean

```diff
@@ -102,7 +102,30 @@ theorem primeCounting'_add_le {a k : ℕ} (h0 : 0 < a) (h1 : a < k) (n : ℕ) :
 theorem zeroth_prime_eq_two : nth Prime 0 = 2 := nth_count prime_two
 
 /-- The `n`th prime is greater or equal to `n + 2`. -/
-lemma add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
+theorem add_two_le_nth_prime (n : ℕ) : n + 2 ≤ nth Prime n :=
   zeroth_prime_eq_two ▸ (nth_strictMono infinite_setOf_prime).add_le_nat n 0
 
+theorem surjective_primeCounting' : Function.Surjective π' := by
```

Comment:
I feel like this should be a specific case of a more specific result, namely, that `Nat.count` is surjective for any predicate satisfied infinitely often / for arbitrarily large values. (`Nat.count_nth_of_infinite` already basically implies this).

Reply:
Yeah, I see what you mean.
I'm using this `Nat.count_nth_of_infinite` already here, but I guess it is a good idea to add the more general form.
Done.